commands.c

```diff
@@ -298,50 +298,285 @@ show_help(int argc, char **argv)
  */
 
+struct ghs arptimeouttab[] = {
+	{ "<seconds>", "Seconds for ARP entries to remain in cache", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs arpdowntab[] = {
+	{ "<seconds>", "Seconds before resending unanswered ARP requests", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs carptab[] = {
+	{ "0", "Disallow CARP", CMPL0 NULL, 0 },
+	{ "1", "Allow CARP", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs carploggingtab[] = {
+	{ "0", "CARP logging priority 0", CMPL0 NULL, 0 },
+	{ "1", "CARP logging priority 1", CMPL0 NULL, 0 },
+	{ "2", "CARP logging priority 2", CMPL0 NULL, 0 },
+	{ "3", "CARP logging priority 3", CMPL0 NULL, 0 },
+	{ "4", "CARP logging priority 4", CMPL0 NULL, 0 },
+	{ "5", "CARP logging priority 5", CMPL0 NULL, 0 },
+	{ "6", "CARP logging priority 6", CMPL0 NULL, 0 },
+	{ "7", "CARP logging priority 7", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs carppreempttab[] = {
+	{ "0", "Disallow virtual CARP hosts to preempt each other", CMPL0 NULL, 0 },
+	{ "1", "Allow virtual CARP hosts to preempt each other", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs forwardingtab[] = {
+	{ "0", "Disable IPv4 Forwarding", CMPL0 NULL, 0 },
+	{ "1", "Enable IPv4 Forwarding", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs mforwardingtab[] = {
+	{ "0", "Disable IPv4 Multicast Forwarding", CMPL0 NULL, 0 },
+	{ "1", "Enable IPv4 Multicast Forwarding", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipiptab[] = {
+	{ "0", "Disallow IP-in-IP Encapsulation", CMPL0 NULL, 0 },
+	{ "1", "Allow IP-in-IP Encapsulation", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs gretab[] = {
+	{ "0", "Disallow Generic Routing Encapsulation", CMPL0 NULL, 0 },
+	{ "1", "Allow Generic Routing Encapsulation", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs wccptab[] = {
+	{ "0", "Disallow Web Cache Control Protocol", CMPL0 NULL, 0 },
+	{ "1", "Allow Web Cache Control Protocol", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs etheriptab[] = {
+	{ "0", "Disallow Ether-IP Encapsulation", CMPL0 NULL, 0 },
+	{ "1", "Allow Ether-IP Encapsulation", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipcomptab[] = {
+	{ "0", "Disallow IP Compression", CMPL0 NULL, 0 },
+	{ "1", "Allow IP Compression", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs esptab[] = {
+	{ "0", "Disallow Encapsulated Security Payload", CMPL0 NULL, 0 },
+	{ "1", "Allow Encapsulated Security Payload", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs espudpencaptab[] = {
+	{ "0", "Disallow ESP encapsulation within UDP", CMPL0 NULL, 0 },
+	{ "1", "Allow ESP encapsulation within UDP", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs espudpencapporttab[] = {
+	{ "<number>", "UDP port number for encapsulation", CMPL0 NULL, 0 },
```

Comment:
line 390 Suggest { "<number>", "UDP port number for ESP encapsulation", CMPL0 NULL, 0 },

Reply:
I would like to keep non-mechanical change out of this if possible.
Granted, I did sneak in one typo fix. But we can change wording separately.